app/i18n/input_i18n.qrc

```diff
@@ -7,6 +7,7 @@
     <file>input_it.qm</file> 
     <file>input_es.qm</file> 
     <file>input_tr.qm</file> 
-    <file>input_fr.qm</file> 
+    <file>input_fr.qm</file>
+    <file>input_en.qm</file> 
```

Comment:
also to info.plist? https://github.com/lutraconsulting/input/blob/master/docs/developers/translations.md#how-to-add-language

Reply:
Hm, I am not sure, there is `en` already.